users.md

```diff
@@ -1,41 +1,69 @@
 ---
-layout: default
+layout: withmap
 title: Cylc Users
 ---
 
 ## Cylc users
 
-Known users include:
-
-  * [National Institute of Water and Atmospheric Research
-    (NIWA)](http://www.niwa.co.nz) - New Zealand
-  * [Met Office](http://www.metoffice.gov.uk) - UK
-  * [Max-Planck-Institut f&uuml;r
-      Meteorologie](http://www.mpimet.mpg.de/mpimet-startseite/) - Germany
-  * [Deutches Klimarechenzentrum (DKRZ)](https://www.dkrz.de/) - Germany
-  * [Bureau of Meteorology](http://www.bom.gov.au/) - Australia
-  * [NRL Marine Meteorology Division](http://www.nrlmry.navy.mil) - USA
-  * [557th Weather Wing](https://www.557weatherwing.af.mil) - USA
-  * [Geophysical Fluid Dynamics Laboratory](http://www.gfdl.noaa.gov/) - USA
-  * [Meteorological Service Singapore (MSS)](http://www.weather.gov.sg/home/) - Singapore
-  * [Phillippine Atmospheric, Geophysical, and Astronomical Services
-    Administration (PAGASA)](http://bagong.pagasa.dost.gov.ph)  - Phillippines
-  * [South African Weather Service (SAWS)](http://www.weathersa.co.za/) - South Africa
-  * [National Centre for Medium Range Weather
-      Forecasting](http://www.ncmrwf.gov.in/) - India
-  * [Korean Meteorological Administration (KMA)](http://web.kma.go.kr/) - Korea
+Major sites known to be using Cylc are shown on the world map above. Sites are
+shown in yellow, with
+[UM Partner](https://www.metoffice.gov.uk/research/approach/collaboration/unified-model/partnership)
+sites being distinguished in green. The sites where Cylc is mainly developed
+are shown in red (note these are also sites within the UM Partnership).
+Organised by geographical area, these sites are:
+
+* Africa:
+  * South Africa:
+    * [South African Weather Service (SAWS)](http://www.weathersa.co.za/)
+* Asia:
+  * India:
+    * [National Centre for Medium Range Weather
+      Forecasting](http://www.ncmrwf.gov.in/)
+  * Korea:
+    * [Korean Meteorological Administration (KMA)](http://web.kma.go.kr/)
+  * Phillippines:
+    * [Phillippine Atmospheric, Geophysical, and Astronomical Services
```

Comment:
s/Phillippine/Philippine

Reply:
These are probably old typos, we can fix them in a future PR (maybe when creating the list from the JS data structure :)

Reply:
Yes probably my fault.  @sadielbartholomew might as well fix them though, now that they've been spotted.